node/node.go

```diff
@@ -1048,7 +1048,15 @@ func (n *Node) initAdminAPI() error {
 	if err != nil {
 		return err
 	}
-	return n.APIServer.AddRoute(service, &sync.RWMutex{}, "admin", "")
+	return n.APIServer.AddRoute(
+		&common.HTTPHandler{
+			LockOptions: common.NoLock,
```

Comment:
This is the main purpose of this PR - converting from `common.WriteLock` to `common.NoLock`.